containers/Dockerfile.hpc.ubuntu.base

```diff
@@ -0,0 +1,13 @@
+FROM ncep_base:ubuntu
```

Comment:
Where is ncep_base:ubuntu stored?

Reply:
@MinsukJi-NOAA 
I was hoping to discuss that with you, @arunchawla-NOAA and whoever this is of interest to at EMC.
I think we can create a dockerhub account for noaa-emc and store these container images there.
If we had a cloud based storage such as AWS:S3 we could store the images there or may be even our FTP site.  I am not sure sure about the last one.
They can be pulled into github actions for CI or for development in a docker container on personal machines.
As we proceed, we can add scripts here to update the images on docker hub (or elsewhere) so it is more or less seamless.

I will add a set of another scripts for Singularity containers in a subsequent PR.

Reply:
Yeah, I briefly looked into Dockerhub.  Individual accounts are free, but organizational accounts are not (https://www.docker.com/pricing).  We could create an "individual account" with name noaa-emc, but not sure if it's the best approach.
I agree that it would be beneficial to have an organizational storage place for docker (and other) images, wherever that may be. AWS is definitely an option. Stored images will be used for the CI, and maybe even for the community by allowing users to download images on a local computer and start running simulations.